src/Anoma/Effect/RunNockma.hs

```diff
@@ -60,5 +55,5 @@ runNockma prog inputs = do
   res :: Response <- anomaRpc runNockGrpcUrl (Aeson.toJSON msg) >>= fromJSON
   logVerbose (mkAnsiText ("Response Payload:\n" <> Aeson.jsonEncodeToPrettyText res))
   case res of
-    ResponseProof x -> decodeJam64 x
+    ResponseProof x -> decodeCue64 x
```

Comment:
Q: `decodeCue64` is correct here because the value of the 'proof' field is jammed - so we cue it to see the nock term?

Reply:
it's just a matter of renaming `decodeJam64` -> `decodeCue64`. They are the same function. I could leave it to `decodeJam64` if that's more intuitive. What do you think?

Reply:
I removed `decodeJam64` in af0dd16827799614ea0e30b911bec87a86475664 because it was a duplicate of `decodeCue64`